temporal-testing/src/main/java/io/temporal/internal/testservice/SelfAdvancingTimerImpl.java

```diff
@@ -181,6 +184,11 @@ public TimerLockHandle(LockEvent event) {
 
     @Override
     public void unlock() {
+      unlock(null);
+    }
+
+    @Override
+    public void unlock(String caller) {
```

Comment:
`caller` is never used? Is it just so it's more obvious during debugging?

Reply:
Yeah. We remove the handle and there is no place to put the message, but I want to kinda promote using this version of the method, so at least you can add a debug printing when you need to investigate what's going on with time locks.